runtime/src/mailbox_api.rs

```diff
@@ -108,6 +108,7 @@ impl MailboxResp {
     pub fn write_to_mbox(&mut self, drivers: &mut Drivers) -> CaliptraResult<()> {
         match self {
             MailboxResp::InvokeDpeCommand(resp) => resp.write_to_mbox(&mut drivers.mbox),
+            MailboxResp::StashMeasurement(resp) => resp.write_to_mbox(&mut drivers.mbox),
```

Comment:
Is our general strategy to keep the runtime mailbox implementation as separate from the ROM ala https://github.com/chipsalliance/caliptra-sw/pull/559 ?

Reply:
Yes. The behavior of this mailbox command is a bit different in runtime. At runtime DPE is initialized so we can just call DeriveChild directly. In ROM we have to stash the measurement so that Runtime can do the necessary DPE operations later.